Hashlink.js

```diff
@@ -137,4 +176,29 @@ export class Hashlink {
     this.registeredTransforms[transform.algorithm] = transform;
   }
 
+  /**
+   * Finds a registered decoder for a given set of bytes or throws an Error.
+   *
+   * @param {Uint8Array} bytes - A stream of bytes to use when matching against
+   *   the registered decoders.
+   */
```

Comment:
Missing `@returns` doc

Reply:
Fixed in d31f47399a9014b2a958d0c7494196a5a7f0bc38.